File: C#/02. Simple Operations and Calculations/Extra/House Painting.cs
using System;

namespace House_Painting
{
    class Program
    {
        static void Main(string[] args)
        {
            double x = double.Parse(Console.ReadLine());
            double y = double.Parse(Console.ReadLine());
            double h = double.Parse(Console.ReadLine());

            double greenArea = 2 * (x * x - 1.2 + x * y - 1.5 * 1.5);
            double redArea = 2 * x * y + x * h;
            double greenPaint = greenArea / 3.4;
            double redPaint = redArea / 4.3;

            Console.WriteLine($"{greenPaint:F2} \n{redPaint:F2}");
        }
    }
}

File: Python/02. Simple Operations and Calculations/Extra/House Painting.py
x = float(input())
y = float(input())
h = float(input())

greenArea = 2 * (x * x - 1.2 + x * y - 1.5 * 1.5)
redArea = 2 * x * y + x * h
greenPaint = greenArea / 3.4
redPaint = redArea / 4.3

print(f"{greenPaint:.2f} \n{redPaint:.2f}")